src/components/Transaction/index.tsx

```diff
@@ -58,18 +118,21 @@ const TransactionComponent = ({
         </div>
         <span className="transaction__separate" />
         <div className="transaction__input__output">
-          <div className="transaction__input">
-            {transaction.display_inputs &&
-              transaction.display_inputs.map((cell: InputOutput) => {
-                return cell && <TransactionCell cell={cell} address={address} key={cell.id} />
-              })}
-          </div>
+          <CellContainerComponent cells={transaction.display_inputs} address={address} pageSize={10} />
           <img src={InputOutputIcon} alt="input and output" />
           <div className="transaction__output">
-            {transaction.display_outputs &&
-              transaction.display_outputs.map((cell: InputOutput) => {
-                return cell && <TransactionCell cell={cell} address={address} key={cell.id} />
-              })}
+            <CellContainerComponent cells={transaction.display_outputs} address={address} pageSize={10} />
+            {address && <span className="transaction__separate" />}
+            {address && confirmation && (
+              <TransactionConfirmationContainer increased={changeInCapacity >= 0}>
+                <div className="confirmation">
+                  {confirmation > 0 ? `${confirmation > 1000 ? '1000+' : confirmation} Confirmation` : `0 Confirmation`}
+                </div>
+                <div className="capacity">
+                  {`${changeInCapacity >= 0 ? '+' : '-'} ${Math.abs(changeInCapacity) / 10 ** 8} CKB`}
```

Comment:
There is a util method for `shannon` and CKB exchanging and we need decimals for CKB, so we don't operate `/ 10**8` directly.

Reply:
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/38/commits/90d38571322909c60039da927d90c3818922c9ca